lua/nvim-treesitter/parsers.lua

```diff
@@ -1041,6 +1041,15 @@ list.idl = {
   maintainers = { "@cathaysia" },
 }
 
+list.idris = {
+  install_info = {
+    url = "https://github.com/kayhide/tree-sitter-idris",
+    files = { "src/parser.c", "src/scanner.c" },
+  },
+  filetype = "idris3",
```

Comment:
```suggestion
  filetype = "idris2",
```

`idris3` is not a recognized filetype.